nocom/src/Account.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './InputBox.css';
+
+function InputBox({ className, onClick }) {
+  const defaultProps = {
+    image: 'https://assets.api.uizard.io/api/cdn/stream/9b130954-67eb-43a8-bd9b-2ef2890b017f.png',
+  };
+
+  return (
+    <button
+      className={`InputBox ${className}`} // Apply the passed className
+      onClick={onClick} // Handle clicks
```

Comment:
it seems on click is not linked to a function

Reply:
> it seems on click is not linked to a function

I am aware and working on it

I only wanted you to see that the location is obscured by the login box since you asked

But I guess we can work on that later

Reply:
good review!!! now you can create issue and reference this review:
<img width="805" alt="Screenshot 2024-11-22 at 21 40 59" src="https://github.com/user-attachments/assets/ef8ef541-41a5-4cf8-9916-eb6994858c71">